lib/search/models/search_result.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:fluttercon/common/data/models/local/local_organiser.dart';
+import 'package:fluttercon/common/data/models/local/local_session.dart';
+import 'package:fluttercon/common/data/models/local/local_speaker.dart';
+import 'package:fluttercon/common/data/models/local/local_sponsor.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'search_result.freezed.dart';
+
+enum SearchResultType { session, speaker, sponsor, organizer }
+
+@freezed
+class SearchResult with _$SearchResult {
+  const factory SearchResult({
+    required String id,
+    required String title,
+    required String subtitle,
+    required String imageUrl,
+    required SearchResultType type,
+    LocalSession? session,
+    LocalSpeaker? speaker,
+    LocalSponsor? sponsor,
+    LocalOrganiser? organizer,
```

Comment:
Instead of `LocalOrganiser` who has no dedicated page, you can have `LocalIndividualOrganiser` instead. The route where you can see the details of an individual organiser is `organiserDetailsRoute`

Ref: `lib/common/utils/router.dart`

Reply:
done